spec/requests/unwithdraw_spec.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 RSpec.describe "Unwithdraw" do
-  let(:edition) { create(:edition, :withdrawn, :political, :past_government) }
-
   describe "POST /documents/:document/unwithdraw" do
     context "when the edition is in history mode" do
+      let(:edition) do
+        create(:edition, :withdrawn, :political, government: past_government)
+      end
```

Comment:
This is now moot as I had to rebase against Peters changes which has outdated this.

I tend to go with trying to have lets near the code they're calling. I don't mind a bit of repetition if it's easy to find where the let is defined. Mainly because I find it difficult to read tests where let definitions occur a long way before a test.